src/legacy/lvm/testing.c

```diff
@@ -335,7 +335,7 @@ control_master_start (void)
   /* Parse the output into a uid */
   g_strstrip (userid->str);
   remote_target_uid = g_ascii_strtoull (userid->str, &endptr, 10);
-  if (!endptr || *endptr != '\0' || remote_target_uid > G_MAXUINT)
```

Comment:
We should probably parse to a guint64, do the range check, and then set the value into ```remote_target_uid```.